app/views/layouts/application.html.erb

```diff
@@ -12,6 +12,7 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
     <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+    <%= favicon_link_tag asset_path('wicilogo.png') %>
```

Comment:
I think Walker used another image for event default image. I'm not sure if we should stay consistent and use the same image in the navbar, because it also follows the colors, or should it be the actual image they gave us?
![Screen Shot 2019-05-12 at 1 44 31 PM](https://user-images.githubusercontent.com/25190615/57587591-36963c80-74bc-11e9-92e3-8b53aa31b504.png)
![Screen Shot 2019-05-12 at 1 44 48 PM](https://user-images.githubusercontent.com/25190615/57587592-36963c80-74bc-11e9-920a-ad8df1497ec8.png)

Reply:
The other image looks weird on the navbar, the grey text is too light and the text on the bottom is unreadable.

Reply:
![image](https://user-images.githubusercontent.com/31327276/57588759-f93aaa80-74cd-11e9-965d-1fb19d48cd56.png)
For comparison

Reply:
![image](https://user-images.githubusercontent.com/31327276/57588767-1a030000-74ce-11e9-9700-b132078802df.png)

Reply:
I like option 2, I think having different images for default isn't a big deal b/c the default image really shouldn't ever show up.